packages/medusa/src/services/__tests__/draft-order.js

```diff
@@ -171,9 +171,8 @@ describe("DraftOrderService", () => {
       expect(lineItemService.generate).toHaveBeenCalledWith(
         [
           {
-            variantId: "test-variant",
+            variant_id: "test-variant",
```

Comment:
todo: this is a breaking change, should we keep the camelcase?

Reply:
if so then I let you check elsewhere

Reply:
but we shouldn't break the API of the generate method either

Reply:
does it make sense to then create a separate method in the line item service like: `generateWithIsolatedProductModule` and call this method when the flag is on?